pkgs/top-level/python-packages.nix

```diff
@@ -4660,6 +4660,11 @@ in {
     inherit (pkgs) pkgconfig;
   };
 
+  wxPython_4_0 = callPackage ../development/python-modules/wxPython/4.0.nix {
+    inherit (pkgs) pkgconfig;
```

Comment:
Why?

Reply:
The reason why we need to explicitly pass things from `pkgs` is because the `callPackage` defined in python-packages.nix
https://github.com/NixOS/nixpkgs/blob/b9a7c07319a249d6d70447e1dcdad9419e61d7cc/pkgs/top-level/python-packages.nix#L23
will only substitute arguments from `self` (aka `pythonPackages`) and not from `pkgs`.

Reply:
pythonPackages seems to have its own pkgconfig attribute too somehow? Meanwhile pkgs has pkg-config... (or I got mixed up somewhere)

Reply:
Ok so pkgs has a pkgconfig and a pkg-config, and pyhtonPackages has a pkgconfig.

Reply:
Okay, so my explanation was wrong. The `callPackage` will also substitute arguments from `pkgs`, but `self` (`pythonPackages`) takes priority. And, indeed we already have `pkgconfig` attribute in `pythonPackages`, but we don't want to get the "pkgconfig" python package, we want just the regular pkgconfig. And this is why we need to explicitly state that it must come from `pkgs`.

Reply:
Correct, you should be able to import something like "tree", even though it doesnt exist in top-level/python-modules.nix, because it inherits from pkgs. However, if you do something like "docker" you'll get `pythonPackages.docker` instead of `./nixpkgs -A docker`